views/js/runner/provider/qti.js

```diff
@@ -400,12 +394,29 @@ define([
                 this.getProbeOverseer().start();
             }
 
-            //load data and current context in parrallel at initialization
+            //load data and current context in parallel at initialization
             return this.getProxy().init()
                        .then(function(results){
+                            var communicator = self.getProxy().getCommunicator();
+
                             self.setTestData(results.testData);
                             self.setTestContext(results.testContext);
                             self.setTestMap(results.testMap);
+
+                            // install the communication channel if enabled
+                            if (communicator) {
```

Comment:
we need to discuss that part. I'm not sure if this should go into the proxy or not...